ApiCheck.Test/Comparer/AssemblyComparerTest.cs

```diff
@@ -28,7 +29,7 @@ public void When_missing_different_types_in_new_assembly_report_should_contain_t
       Assembly assembly1 = ApiBuilder.CreateApi()
         .Class("Class1").Build()
         .AbstractClass("Abstract1").Build()
-        .Enum("Enum1")
+        .Enum("Enum1", new[] { Tuple.Create("Value1", 1) })
```

Comment:
I've updated this to avoid the ambiguity between ApiBuilder Enum overloades.

Reply:
Can be avoided by providing a generic `ApiBuilder.Enum<T>` overload.

Reply:
Yeah, It's much cleaner. Thank you for the suggestion.

Reply:
I've resolved it.